commands/deploy.go

```diff
@@ -299,6 +299,11 @@ func getGatewayURL(argumentURL string, defaultURL string, yamlURL string) string
 		gatewayURL = defaultURL
 	}
 
+	gatewayURL = strings.ToLower(strings.TrimRight(gatewayURL, "/"))
```

Comment:
What about `https://`?

Reply:
I don't think it will pick up https.. https will always need to be typed in separately but this should make testing quicker for us when we are on localhost etc and forget to add `http://` before `localhost:8080`